### PR TITLE
updating rendering functionality

### PR DESCRIPTION
**Pull request by alexmcraek**
making it look like this:

image

### CODE REVIEW

File: finmodel/src/render.py [hunk 1]
```diff
@@ -20,15 +20,18 @@
...
     df.reset_index(inplace=True, drop=False)
     df.rename(columns={"Account": "ACCOUNT",
                        "Year": "YEAR",
                        "yoy_growth": "YOY\nGROWTH",
                        "gizmo_mix": "GIZMO\nMIX",
+                       "Source": "SOURCE",
+                       "Sales": "SALES\n(USD)",
                         }, inplace=True)
-
     df.fillna(" ", inplace=True)
```
Comment (tray3j) [inline finmodel/src/render.py:35]: I think this line was throwing a warning for me, didn't like the str with an int64 or float64 array

File: finmodel/src/render.py [hunk 2]
```diff
@@ -74,24 +91,30 @@ def render_df(df):
     # -----------------------------------------------------------------------------------------------------------------------------------------------
     # Formatting column widths and justifications
     for column in columns:
-        if column == "YOY\nGROWTH" or column == "GIZMO\nMIX":
+        if column == "YOY\nGROWTH" or column == "GIZMO\nMIX" or column == "SALES\n(USD)":
```
Comment (tray3j) [inline finmodel/src/render.py:94]: # TODO: make this dynamic or an arg in the render func call

**Review by tray3j — APPROVED** [review]

File: finmodel/src/render.py (hunk 1 above)
Comment (alexmcraek) [reply finmodel/src/render.py:35]: I am suppressing the warning for now on line 26. It's just cause im going in an messing with it purely for rendering purposes. I will find a different fix later.

File: finmodel/src/render.py (hunk 2 above)
Comment (alexmcraek) [reply finmodel/src/render.py:94]: in other words, we should write a rendering class.

**Pull request merged**